pkg/changetracking/changetracking_types.go

```diff
@@ -0,0 +1,38 @@
+// Code generated by tutone: DO NOT EDIT
+package changetracking
+
+import (
+	"github.com/newrelic/newrelic-client-go/v2/pkg/common"
+	"github.com/newrelic/newrelic-client-go/v2/pkg/nrtime"
+)
+
+/*
+Defer ChangeTrackingDeploymentInput placement into types.go
+until CustomAttributes GA.
```

Comment:
```suggestion
Since we can't yet generate the CustomAttributes type with Tutone, 
defer ChangeTrackingDeploymentInput placement into types.go until CustomAttributes GA.
```